packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/withdrawPayment.ts

```diff
@@ -31,6 +32,28 @@ const validateInput = (input: CommandInput): boolean => {
   return true
 }
 
+const beforeExecute: BeforeExecute<CommandInput, ContractInput> = (context) => async () => {
+  logger.info(`Transmitter ${context.contractInput.transmitter} withdrawing LINK payment from ${context.contract}`)
+
+  await prompt('Continue?')
+  return
+}
+
+const afterExecute = () => async (response: Result<TransactionResponse>) => {
+  const events = response.responses[0].tx.events
+  if (!events) {
+    logger.error('Could not retrieve events from tx')
+    return
+  }
+
+  const transmitter = events[0].wasm.transmitter[0]
+  const payee = events[0].wasm.payee[0]
+  const amount = events[0].wasm.amount[0]
+  const link_token = events[0].wasm.link_token[0]
```

Comment:
This is the basicallty `oracle_paid` (https://github.com/smartcontractkit/chainlink-terra/blob/develop/contracts/ocr2/src/contract.rs#L1320) event information. We could start a `/lib/events.ts` section where we can type all these events:
```ts
// /lib/events.ts
type EventOraclePaid = {
  transmitter: string
  payee: string
  amount: string
  linkToken: string
  method?: string
}

export const parseOraclePaidEvent = (event): EventOraclePaid => {
   // Parse and validate every value
}

// withdrawPayment.ts
const afterExecute = () => async (response: Result<TransactionResponse>) => {
  // ...
  const paidOracleEvent = parseOraclePaidEvent(events[0].wasm)
  // ...
}
```